bitmap.h

```diff
@@ -5,22 +5,27 @@
 #include "simplefs.h"
 
 /*
- * Return the first free bit (set to 1) in a given in-memory bitmap spanning
- * over multiple blocks and clear it.
- * Return 0 if no free bit found (we assume that the first bit is never free
- * because of the superblock and the root inode, thus allowing us to use 0 as an
- * error value).
+ * Return the first `len` free bits (set to 1) in a given in-memory bitmap
+ * spanning over multiple blocks and clear it. Return 0 if no enough free bits
+ * were found (we assume that the first bit is never free because of the
+ * superblock and the root inode, thus allowing us to use 0 as an error value).
  */
-static inline uint32_t get_first_free_bit(unsigned long *freemap,
-                                          unsigned long size)
+static inline uint32_t get_first_free_bits(unsigned long *freemap,
+                                           unsigned long size,
+                                           uint32_t len)
 {
-    uint32_t ino = find_first_bit(freemap, size);
-    if (ino == size)
-        return 0;
-
-    bitmap_clear(freemap, ino, 1);
-
-    return ino;
+    uint32_t bit, prev = 0, count = 0;
+    for_each_set_bit(bit, freemap, size)
```

Comment:
Append the following lines into file `.clang-format`:
```
ForEachMacros:
  - 'for_each_set_bit'
  - 'for_each_set_bit_from'
```

Then, use `clang-format -i` to indent.

Reply:
I have modified it, thanks！